docker/ci/dockerfiles/current/docker-builder.ubuntu2004.x64.dockerfile

```diff
@@ -54,3 +56,9 @@ RUN curl -L https://github.com/google/go-containerregistry/releases/latest/downl
     chmod +x gcrane && \
     mv gcrane /usr/local/bin/ && \
     rm -rf go-containerregistry.tar.gz 
+
+# Install gh-cli
```

Comment:
What do we need gh-cli for?

Reply:
Basically every image now have gh-cli for creating issues on GitHub. @rishabh6788 

I am just adding this up which does not take too much space.

If you think it is not needed we can always remove.

Reply:
Looks like the post actions runs on agent node directly https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L879
@rishabh6788 can add more. Remove if not required.

Reply:
Yes, it is run on the node directly, not required inside the docker image. @peterzhuamazon @gaiksaya

Reply:
This is more of a per-requisite in case we need to use this for gh cli.

Reply:
Removed now.